docs/conversion_examples_gallery/imaging/scanimage.rst

```diff
@@ -20,9 +20,16 @@ Convert ScanImage imaging data to NWB using
     >>> from pathlib import Path
     >>> from neuroconv.datainterfaces import ScanImageImagingInterface
     >>>
-    >>> file_path = OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"
-    >>> interface = ScanImageImagingInterface(file_path=file_path, channel_name="Channel 1", plane_name="0", verbose=False)
     >>>
+    >>> file_path = str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "volumetric_single_channel_single_file_no_flyback" / "vol_no_flyback_00001_00001.tif")
+    >>>
+    >>> # Specify channel_name for multi-channel data
+    >>> # Specify plane_index for selecting a specific plane in multi-plane data or leave undefined  to write volumetric data
+    >>> interface = ScanImageImagingInterface(
+    ...     file_path=file_path,
+    ...     channel_name="Channel 1",  # Required for multi-channel data
+    ...     plane_index=None,  # Optional: specify to only write a specific plane
+    ... )
```

Comment:
Maybe having an example to demonstrate when to use `interleave_slice_samples` or `slice_sample`?

Reply:
Thanks for explaining the changes, I this this comment can be safely ignored here and maybe addressed when you're working on the documentation of writing methods

Reply:
I think I would like to add more thorough documentation in another PR indicating how to achieve workflows A, B and C here:

https://github.com/catalystneuro/neuroconv/issues/1260